src/lib/Server/Values/RestFieldDefinition.php

```diff
@@ -30,14 +30,16 @@ class RestFieldDefinition extends RestValue
     public $fieldDefinition;
 
     /**
-     * Construct.
+     * Path which is used to fetch the list of field definitions.
      *
-     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
-     * @param \eZ\Publish\API\Repository\Values\ContentType\FieldDefinition $fieldDefinition
+     * @var string|null
      */
-    public function __construct(ContentType $contentType, FieldDefinition $fieldDefinition)
+    public $path;
```

Comment:
property type is missing in phpdoc

Reply:
`@var string|null`
I think it's correct, it can be string or null, unless I missed something?

Reply:
`@var string|null`
I think it's correct, it can be string or null unless I missed something.

Reply:
Sorry, I've missed `@var string|null` in diff